src/containerapp/azext_containerapp/_help.py

```diff
@@ -470,7 +517,7 @@
 
 helps['containerapp dapr enable'] = """
     type: command
-    short-summary: Enable Dapr for a container app.
+    short-summary: Enable Dapr for a container app. Updates existing values.
```

Comment:
Curious given we added we update existing values here. What happens with disable? Do we remove existing values?